.changeset/sixty-numbers-reply.md

```diff
@@ -0,0 +1,5 @@
+---
+'openzeppelin-solidity': major
+---
+
+`Governor`: Add a `voter` parameter to the `castVoteBySig` and `castVoteWithReasonAndParamsBySig` functions to avoid forging signatures for random addresses and allow the use of an internal nonce for invalidating signatures.
```

Comment:
Mostly highlighting what I think is most important about this change:
```suggestion
`Governor`: Add `voter` and `nonce` parameters in signed ballots, to avoid forging signatures for random addresses, prevent signature replay, and allow invalidating signatures. Add `voter` as a new parameter in the `castVoteBySig` and `castVoteWithReasonAndParamsBySig` functions.
```